src/ostree/ot-builtin-summary.c

```diff
@@ -55,10 +109,16 @@ ostree_builtin_summary (int argc, char **argv, GCancellable *cancellable, GError
 
   if (opt_update)
     {
+      g_autoptr(GVariant) additional_metadata = NULL;
+
       if (!ostree_ensure_repo_writable (repo, error))
         goto out;
 
-      if (!ostree_repo_regenerate_summary (repo, NULL, cancellable, error))
+      additional_metadata = argv_to_additional_metadata (&argc, &argv, error);
```

Comment:
A bit UX bikeshed, though any reason to use additional args for this? Seems random vs. a more explicit e.g. `--add-metadata` (repeatable) option.